assets/src/edit-story/components/form/dateTime/calendarStyle.js

```diff
@@ -0,0 +1,86 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable header/header */
+/* Disable reason: This file is based on Gutenberg Datepicker */
+
+/**
+ * External dependencies
+ */
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+.CalendarMonth_caption {
+  font-size: 13px;
+}
+.CalendarDay {
+  font-size: 13px;
+  border: 1px solid transparent;
+  border-radius: 50%;
+  text-align: center;
+}
+.CalendarDay__default:hover {
+  border: 1px solid transparent;
+}
+.CalendarDay__selected {
+  background: #0085ba;
+}
+.CalendarDay__selected:hover {
+  background: rgb(0, 113, 158);
+}
+body.admin-color-sunrise .CalendarDay__selected {
```

Comment:
Should we ignore these and perhaps choose a static color that fits our editor instead?